client/src/components/Hours.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Hours extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      hoursToggle: true,
+    };
+  }
+
+  handleHoursToggle(e) {
+    e.preventDefault();
+    const { hoursToggle } = this.state;
```

Comment:
Don't need to destructure this here

Reply:
The linter is making me de-structuring it!